src/palace/manager/celery/tasks/work.py

```diff
@@ -32,10 +32,9 @@ def calculate_work_presentations(
         work_policies = waiting.pop(batch_size)
 
         if work_policies:
-
             try:
+                remaining_work_policies = work_policies.copy()
```

Comment:
Minor: Given that we are processing one per session now, we lose the advantage of batching. I'm not sure there is much reason to batching these in a Redis set. Doesn't have to be part of this PR, but seems like we might want to just process them in a task one at a time.